.github/workflows/dist.yml

```diff
@@ -16,6 +16,51 @@ jobs:
       run: |
         pip install black
         black --check --diff .
+  
+  build:
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: [windows-latest, macos-latest, ubuntu-18.04]
+        python_version: [3.6, 3.7, 3.8]
+        architecture: [x86, x64]
+        exclude:
+        - os: macos-latest
+          architecture: x86
+        - os: ubuntu-18.04
+          architecture: x86
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: Checkout submodules
+      shell: bash
+      run: |
+        auth_header="$(git config --local --get http.https://github.com/.extraheader)"
+        git submodule sync --recursive
+        git -c "http.extraheader=$auth_header" -c protocol.version=2 submodule update --init --force --recursive --depth=1
+    
+    - uses: actions/setup-python@v1
+      with:
+        python-version: ${{ matrix.python_version }}
+        architecture: ${{ matrix.architecture }}
+
+    - name: Install build dependencies
+      run: pip install wheel
+
+    - name: Build wheel
+      run: python setup.py bdist_wheel
+    
+    - name: Test wheel
+      shell: bash
+      env:
+        RPYBUILD_PARALLEL: 1
+        RPYBUILD_STRIP_LIBPYTHON: 1
+      run: |
+        cd dist
+        python -m pip install *.whl
+        cd ../tests
+        python -m pip install -r requirements.txt
+        python run_tests.py
 
   publish:
     runs-on: ubuntu-latest
```

Comment:
Can't comment on the next line, but you probably want to depend on the job you're adding here too.